wandb/deploys/launch-agent.yaml

```diff
@@ -0,0 +1,86 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: wandb-launch-agent
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: launch-agent-demo
+  namespace: wandb-launch-agent
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: launch-agent-demo
+  template:
+    metadata:
+      labels:
+        app: launch-agent-demo
+    spec:
+      securityContext:
+        fsGroup: 107    # Group ID of docker group on k8s nodes. Also 412
+      containers:
+      - name: launch-agent-demo
+        image: launch-agent-container:v0
+        resources:
+          limits:
+            memory: "2Gi"
+            cpu: "1000m"
+        env:
+        - name: WANDB_PROJECT
+          valueFrom:
+            configMapKeyRef:
+              name: wandb-setup
+              key: wandb-project
+        - name: WANDB_BASE_URL
+          valueFrom:
+            configMapKeyRef:
+              name: wandb-setup
+              key: wandb-base-url
+        - name: WANDB_API_KEY
+          valueFrom:
+            secretKeyRef:
+              name: wandb-api-key
+              key: password
+        securityContext:
+          privileged: true
+          runAsNonRoot: false # added
+          runAsUser: 0
+          capabilities: 
+            drop:
+              - all
+          allowPrivilegeEscalation: true
+        volumeMounts:
+        - name: dockersock
+          mountPath: "/var/run/docker.sock"
+        # - name: wandb-dir
+        #   mountPath: "/root/wandb"
+      volumes:
+      - name: dockersock
+        hostPath:
+          path: /var/run/docker.sock
```

Comment:
Do we know if gong even has a docker socket in their k8s cluster?  GKE has moved [to containerd](https://kubernetes.io/blog/2020/12/02/dont-panic-kubernetes-and-docker/) and I think EKS and AKS are doing the same which would mean there's no docker socket to mount...

Reply:
We know that they are using containers running the docker runtime. But, AWS has made DooD more difficult as of Jan, so we're actually abandoning the DooD aspect of this and this needs to be removed.

Reply:
The actual deployment YAML will be entirely changed by Jasons PR which will go in after this one. I'll remove this file entirely